### PR TITLE
Support for JsonLdRenderer in RDFView

### PR DESCRIPTION
**Pull request by tijmenbaarda**
`RDFView` was already able to support `JsonLdRenderer`, but it was not possible to pass a context to RDFLib. Allow passing a context and mention support for `JsonLdRenderer` in the docstring.

The way I implemented this may seem a bit clumsy, but it was the cleanest solution I could come up with -- there might be a better way.

### CODE REVIEW

File: rdf/renderers.py [hunk 1]
```diff
@@ -22,7 +22,14 @@ class RDFLibRenderer(BaseRenderer):
...
     def render(self, graph, media_type=None, renderer_context=None):
-        return graph.serialize(**self.rdflib_args)
+        view = None
+        if renderer_context is not None and 'view' in renderer_context:
+            view = renderer_context['view']
+        rdflib_args = self.get_rdflib_args(view=view)
+        return graph.serialize(**rdflib_args)
+
+    def get_rdflib_args(self, **kwargs):
+        return self.rdflib_args
```
Comment (jgonggrijp) [inline rdf/renderers.py:32]: Is this what you were referring to as "clumsy-looking"? In that case I see why you chose to use those words, but I also see why you did it this way. It looks reasonable to me.

File: rdf/renderers_test.py [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+    view.json_ld_context = None
+    renderer = JsonLdRenderer()
+    context = {"view": view}
+    serialization = renderer.render(filled_graph, renderer_context=context)
+    assert verify_serialization(serialization, filled_graph, "json-ld")
+
+
+def test_jsonldrenderer_with_jsonld_context(filled_graph):
+    # Note that this test does not check if the JSON-LD context is correctly
+    # applied - we assume that rdflib does this correctly.
```
Comment (jgonggrijp) [inline rdf/renderers_test.py:41]: I agree with this approach in general, although rdflib has failed us in the past.

No need to add more test code, but for peace of mind: did you eyeball rdflib's JSON-LD output to check that it *seemed* correct?

**Review by jgonggrijp — APPROVED** [review]
Looks good to me, just one question.

File: rdf/renderers_test.py (hunk 2 above)
Comment (tijmenbaarda) [reply rdf/renderers_test.py:41]: Yes :)

**Comment by tijmenbaarda** [comment]
I forgot that this PR was still open after approval. I will merge it and create a release.

**Comment by tijmenbaarda** [comment]
> I forgot that this PR was still open after approval. I will merge it and create a release.

... I see that this library already has a stable version. I think this can be added in a minor release, since this change is not supposed to break anything?

**Pull request merged**